### ISSUE TITLE
Could you also provide the BPE version of the TF LM example?

### ISSUE THREAD

**Issue by cloudygoose** [pytorch/fairseq#1251]
Hi,
For the provided trained LM in
https://github.com/pytorch/fairseq/tree/master/examples/language_model/transformer_lm
I downloaded it and find that the vocabulary is word-level
In the paper BPE-level model is also discussed, could you release that trained model as well?
Thanks!

**Comment by huihuifan** [comment]
@alexeib

**Comment by myleott** [comment]
Make sure you're training on enough GPUs so that the batch size is appropriate. See Section 4.3 of the original paper for more details: https://arxiv.org/pdf/1809.10853.pdf

If you are training on a single GPU then you will need to accumulate gradients to achieve a larger effective batch size. You can do this by passing `--update-freq=16`, which will simulate training on 16x as many GPUs.
